docs/source/tutorial/commercial_paper.md

```diff
@@ -450,135 +357,167 @@ Feel free to examine other files in the `contract` directory to understand how
-and
-[getState()](https://hyperledger.github.io/fabric-chaincode-node/master/api/fabric-shim.ChaincodeStub.html#getState__anchor)
-Fabric APIs. Examine how these APIs are used by `StateList` class within
-`ledger-api\statelist.js`.
-
-Let's now install `papercontract` as the MagnetoCorp administrator. In the
-MagnetoCorp administrator's command window, use the `docker exec` command to run
-the `peer chaincode install` command in the `cliMagnetCorp` container:
+### Install and approve the smart contract as MagnetoCorp
 
+We will first install and approve the smart contract as the MagnetoCorp admin. Make
+sure that you are operating from the `magnetocorp` folder, or navigate back to that
+folder using the following command:
 ```
-(magnetocorp admin)$ docker exec cliMagnetoCorp peer chaincode install -n papercontract -v 0 -p /opt/gopath/src/github.com/contract -l node
-
-2018-11-07 14:21:48.400 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 001 Using default escc
-2018-11-07 14:21:48.400 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default vscc
-2018-11-07 14:21:48.466 UTC [chaincodeCmd] install -> INFO 003 Installed remotely response:<status:200 payload:"OK" >
+cd commercial-paper/organization/magnetocorp
 ```
 
-The `cliMagnetCorp` container has set
-`CORE_PEER_ADDRESS=peer0.org1.example.com:7051` to target its commands to
-`peer0.org1.example.com`, and the `INFO 003 Installed remotely...` indicates
-`papercontract` has been successfully installed on this peer. Currently, the
-MagnetoCorp administrator only has to install a copy of `papercontract` on a
-single MagentoCorp peer.
+A MagnetoCorp administrator can interact with PaperNet using the `peer` CLI. However,
+the administrator needs to set certain environment variables in their command
+window to use the correct set of `peer` binaries, send commands to the address
+of the MagnetoCorp peer, and sign requests with the correct crypto material.
 
-Note how `peer chaincode install` command specified the smart contract path,
-`-p`, relative to the `cliMagnetoCorp` container's file system:
-`/opt/gopath/src/github.com/contract`. This path has been mapped to the local
-file system path `.../organization/magnetocorp/contract` via the
-`magnetocorp/configuration/cli/docker-compose.yml` file:
-
-```yaml
-volumes:
-    - ...
-    - ./../../../../organization/magnetocorp:/opt/gopath/src/github.com/
-    - ...
+You can use a script provided by the tutorial to set the environment variables in
+your command window. Run the following command in the `magnetocorp` directory:
+```
+source magnetocorp.sh
 ```
 
-See how the `volume` directive maps `organization/magnetocorp` to
-`/opt/gopath/src/github.com/` providing this container access to your local file
-system where MagnetoCorp's copy of the `papercontract` smart contract is held.
+You will see the full list of environment variables printed in your window. We
```

Comment:
The 'source' command will run the script in the context of your current command line shell - so it won't output the values, but attempt to set them directory.  

./magnetocorp.sh will by itself output the commands to use, and
source magentocorp.sh will set the values in your current shell.

Though as Nik found out, there are slight variations in different shells used by different Linux/Mac etc... so the 'source command might not work.  eg I had to use source <(./magnetocorp.sh)